src/main/java/org/folio/qm/client/LinksClient.java

```diff
@@ -40,8 +52,28 @@ class InstanceLink {
     UUID authorityId;
     String authorityNaturalId;
     UUID instanceId;
-    String bibRecordTag;
-    List<String> bibRecordSubfields;
     Integer linkingRuleId;
   }
+
+  @Data
+  class LinkingRuleDto {
+    private Integer id;
+    private String bibField;
+    private String authorityField;
+    private List<String> authoritySubfields = new ArrayList();
+    private List<SubfieldModification> subfieldModifications = new ArrayList();
+    private String bibRecordTag;
+
+    private SubfieldValidation validation;
+  }
+
+  class SubfieldModification {
+    private String source;
+    private String target;
+    private Map<String, Object> additionalProperties = new HashMap<String, Object>();
+  }
+
+  class SubfieldValidation {
+    private List<Map<String, Boolean>> existence = null;
+  }
 }
```

Comment:
I think we do not need all these fields. You can leave only fields that are needed for implementation